src/main/resources/interface-specification.yml

```diff
@@ -6873,6 +6873,80 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /clients/api:
```

Comment:
same considerations [here](https://github.com/pagopa/interop-be-backend-for-frontend/pull/147#discussion_r1135158659)